java/src/jmri/jmrit/roster/PrintRosterEntry.java

```diff
@@ -167,6 +179,15 @@ public boolean isBusy() {
         return false;
     }
 
+    /**
+     * Configure variable fields in class.
+     *
+     * @param rosterEntry an item in the Roster
+     * @param paneList list of programmer tabs
+     * @param flPane pane w/checkbox to select printing of "Function List"
+     * @param rMPane pane containing roster media (image)
+     * @param parent Roster JFrame calling this method
```

Comment:
The parent parameter is not necessarily a Roster JFrame, or the caller of this method; its the window over which this dialog will be centered.

BTW, thank you for adding Javadoc comments. Its very helpful.